src/views/Project/Details/WordCategories/WordCategoryDetail/CloneWordCategoryButton/requests/WordCategoryCloneRequest.js

```diff
@@ -0,0 +1,41 @@
+import {
+    createUrlForCeClone,
+    createParamsForCeClone,
+} from '#rest';
+import Request from '#utils/Request';
+
+export default class ProjectCeCloneRequest extends Request {
+    schemaName = 'categoryEditor'
+
+    handlePreLoad = () => {
+        this.parent.setState({ pendingWordCategoryClone: true });
+    }
+
+    handlePostLoad = () => {
+        this.parent.setState({ pendingWordCategoryClone: false });
+    }
+
+    handleSuccess = (response) => {
+        this.parent.setState({ showCloneWordCategoryModal: false });
```

Comment:
Data should be cleared on unmount here too.

Reply:
done